locales/fr.json

```
@@ -16,12 +16,12 @@
     "operation_interrupted": "Opération interrompue",
     "password": "Mot de passe",
     "pattern_not_match": "Ne correspond pas au motif",
-    "root_required": "Vous devez être super-utilisateur pour exécuter cette action",
+    "root_required": "Vous devez être admin pour exécuter cette action",
```

```suggestion
    "root_required": "Vous devez avoir les droits d'administration pour exécuter cette action",
```

(recopypasting because weblate force-pushed since then)